dont_takeoff: False
print_kalman: False
print_controller: True

theta_trim: -1.8768106841432017
phi_trim: 0.2882582418517803

uri: "radio://0/50/2M/E7E7E7E708"
